data_integration/config.py

```diff
@@ -72,15 +72,23 @@ def slack_token() -> typing.Optional[str]:
     """
     return None
 
+def teams_token() -> str:
+    """
+    When not None, then this teams webhook is notified of failed nodes.
+    """
+    return None
+
 @functools.lru_cache(maxsize=None)
 def event_handlers() -> [events.EventHandler]:
-    """User specific envent handlers (mainly to notify chat systems)"""
-    configured_handlers = []
+    from .notification import slack, teams, notifier
```

Comment:
importing slack and teams here will mean any import time work will be done regardless if it is configured or not. This should be fine (it's just defining a small class) but IMO it's cleaner to import only if they will be used.

Reply:
Yes, done.